CAM Platform/api-providers.md

```diff
@@ -3446,118 +1500,125 @@ ElasticBox-Release: 4.0
 
-* Invalid Data (400)
-* Forbidden (403)
-* Not Found (404)
+### Request
+#### Headers
+```
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
 
-**Request Parameters**
+#### URI Parameters
+| Parameter | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Provider id | string | The unique id that given to each provider | Yes |
 
+#### Request Body Parameters
 |Parameter | Type | Description|
 |----------|------|------------|
 |location | string | Image location. |
 |name | string | Image name. |
 |description | string | Image description. |
 
+#### Request Body
 ```
-Headers:
-
-Content-Type: application/json
-Authorization: Bearer your_json_web_token
-ElasticBox-Release: 4.0
-```
-
-```
-Body:
-
 {
    "location":"us-east-1",
    "name":"ami-3275ee5b",
    "description":"New Image Description"
 }
 ```
 
-**DELETE /services/providers/{provider_id}/images/{machine_image_id}**
-
-Deletes an existing machine image when you give the provider ID and the machine image ID.
+### Response
+#### Normal Response Codes
+- **202** Accepted
 
-**Normal Response Codes**
+#### Common Error Response Codes
+- 400: Bad Request - Request missing, incomplete or includes invalid properties (details provided inside body)
+- 403: Forbidden - User doesn’t belong to the organization
+- 404: Not Found
 
-* 202
 
-**Error Response Codes**
+## DELETE /services/providers/{provider_id}/images/{machine_image_id}
 
-* Location query parameter is missing (400)
-* Forbidden (403)
-* Not Found (404)
+Deletes an existing machine image when you give the provider ID and the machine image ID.
 
-**Request Parameters**
+### URL
+#### Structure
+    [DELETE] /services/providers/{provider_id}/images/{machine_image_id}
 
-| Parameter | Type | Description |
-|-----------|------|-------------|
-|location | string | Location of the machine image to be deleted. |
+#### Example
+    DELETE https://cam.ctl.io/services/providers/338f38dc-e667-47e0-9026-b253138f109e/images/b253138f109e?location=us-east-1
 
+### Request
+#### Headers
 ```
-Headers:
-
 Content-Type: application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
 
-```
-Body:
+#### URI Parameters
+| Parameter | Type | Description | Req. |
+| --- | --- | --- | --- |
+| Provider id | string | The unique id that given to each provider | Yes |
 
-DELETE /services/providers/{provider_id}/images/{machine_image_id}?location=us-east-1
-```
+#### Request Body Parameters
+| Parameter | Type | Description |
+|-----------|------|-------------|
+|location | string | Location of the machine image to be deleted. |
+
+
+### Response
+#### Normal Response Codes
+- **202** Accepted
+
+#### Common Error Response Codes
+- 400: Bad Request - Request missing, incomplete or includes invalid properties (details provided inside body)
+- 403: Forbidden - User doesn’t belong to the organization
+- 404: Not Found
 
```

Comment:
There is one additional endpoint for searching for images: `/providers/<provider_id>/search/images`; please take a look and document.

Reply:
That was probably a temporary error in beta (there were many issues thursday/friday last week); give it a try now. Only Azure ARM supports this endpoint.